test/unit/forms/declaration/IsExsSpec.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.forms.declaration
+
+import forms.declaration.IsExs
+import play.api.data.FormError
+import play.api.libs.json.{JsObject, JsString}
+import unit.base.UnitSpec
+
+class IsExsSpec extends UnitSpec {
+
+  "Is Exs" should {
+
+    "has a correct key" in {
+
+      IsExs.isExsKey mustBe "isExs"
+    }
+  }
+
+  "Is Exs form" should {
+
+    "has not errors" when {
+
+      "the answer is Yes" in {
+
+        IsExs.form.fillAndValidate(IsExs("Yes")).errors mustBe empty
+      }
+
+      "the answer is No" in {
+
+        IsExs.form.fillAndValidate(IsExs("No")).errors mustBe empty
+      }
+    }
+  }
```

Comment:
This curly brace should be moved to the end of the file. Test names are incorrect now.

Reply:
Moved